content/applications/inventory_and_mrp/inventory/product_management/configure.rst

```diff
@@ -1,9 +1,167 @@
-:nosearch:
+:show-content:
+:hide-page-toc:
```

Comment:
I recommend showing the _on page TOC_ (not to be confused with the es.html#document-metadata)
```suggestion
```